src/spikeinterface/core/base.py

```diff
@@ -223,12 +229,14 @@ def set_property(self, key, values: Sequence, ids: Optional[Sequence] = None, mi
             Array of values for the property
         ids : list/np.array, default: None
             List of subset of ids to set the values, default: None
+            if None which is the default all the ids are set or changed
         missing_value : object, default: None
             In case the property is set on a subset of values ("ids" not None),
             it specifies the how the missing values should be filled.
             The missing_value has to be specified for types int and unsigned int.
         """
 
+        # This deletes the values but we have `delete_property` maybe we should eliminate this?
```

Comment:
@alejoe91

Reply:
Thanks @h-mayorquin 

Yeah I think we should remove this weird behavior and suggest to use `delete_property`

Reply:
Great! Doing it now.

Reply:
Done.

Reply:
This is actually used (I don't think that intentionally)

https://github.com/h-mayorquin/spikeinterface/blob/7b37ee3f45abe56debc6f05d2131f3114e94c10f/src/spikeinterface/core/base.py#L1138-L1140

But I would rather address that into another PR.